sections/browsers.include

```diff
@@ -59,28 +59,27 @@
   <a>creator browsing context</a>. Otherwise, the <a>browsing context</a> has no
   <a>creator browsing context</a>.
 
-  <p>If a <span>browsing context</span> <var>context</var> has a <span>creator browsing
-  context</span> <var>creator</var>, it also has the following properties. In what follows, let
-  <var>creator document</var> be <var>creator</var>'s <span>active document</span> at the time
+  <p>If a [=browsing context=] <var>context</var> has a [=creator browsing context=]
+  <var>creator</var>, it also has the following properties. In what follows, let
+  <var>creator document</var> be <var>creator</var>'s [=active document=] at the time
   <var>context</var> is created:</p>
 
   <dl>
    <dt><dfn>creator origin</dfn></dt>
-   <dd><var>creator document</var>'s <span>origin</span></dd>
+   <dd>|creator document|'s <a for="/">origin</a></dd>
 
    <dt><dfn>creator URL</dfn></dt>
-   <dd><var>creator document</var>'s <span data-x="concept-document-url">URL</span></dd>
+   <dd>|creator document|'s <a data-x="concept-document-url" for="/">URL</a></dd>
 
    <dt><dfn>creator base URL</dfn></dt>
-   <dd><var>creator document</var>'s <span data-x="document base URL">base URL</span></dd>
+   <dd>|creator document|'s <a data-x="document base URL">base URL</a></dd>
 
    <dt><dfn>creator referrer policy</dfn></dt>
-   <dd><var>creator document</var>'s <span data-x="concept-document-referrer-policy">referrer
-   policy</span></dd>
+   <dd>|creator document|'s <a data-x="concept-document-referrer-policy">referrer policy</a></dd>
 
    <dt><dfn>creator context security</dfn></dt>
    <dd>The result of executing <span>Is environment settings object a secure context?</span> on
-   <var>creator document</var>'s <span>relevant settings object</span></dd>
+   |creator document|'s <span>relevant settings object</span></dd>
```

Comment:
That `<span>` should be an `<a>`. In general, whenever you see a `<span>` question it. `<span>` is used by Anolis for linking (which is what the WHATWG source code uses). For Bikeshed, it must be an `<a>` (or any of the other flavors of autolinking it supports).